src/trix/elements/trix_editor_element.js

```diff
@@ -228,7 +228,7 @@ class ElementInternalsDelegate {
     const { required, value } = this.element
     const valueMissing = required && !value
     const customError = !!customValidationMessage
-    const input = Object.assign(document.createElement("input"), { required })
+    const input = makeElement("input", { required })
```

Comment:
@jorgemanrubia I'm not sure how to make sure it passes CI on `main`, but I _do_ think this is worth merging on its own merits, since it re-uses an existing helper instead of an ad hoc combination of `Object.assign` and `Document.createElement`.